app/views/declaration/summary/package_information.scala.html

```diff
@@ -22,24 +22,26 @@
 @(itemId: String, itemNo: Int, packageInformation: Seq[PackageInformation])(implicit messages: Messages, journeyRequest: JourneyRequest[_])
 
 @if(packageInformation.nonEmpty){
+  <table>
```

Comment:
I know that this `table` will solve the problem with `caption`, but now we will have a nested tables. Is it okay? I mean this component is used inside another table

Reply:
It actually solves the problem with the Change link alignment. 
The summary page was not accessible, with this change we introduce three new alerts